packages/infrastructure/kube_airbyte/vars.tf

```diff
@@ -0,0 +1,350 @@
+variable "namespace" {
+  description = "The namespace to deploy Airbyte into"
+  type        = string
+  default     = "airbyte"
+}
+
+variable "airbyte_edition" {
+  description = "The edition of Airbyte to deploy (community or enterprise)"
+  type        = string
+  default     = "community"
+  validation {
+    condition     = contains(["community", "enterprise"], var.airbyte_edition)
+    error_message = "The airbyte_edition value must be either 'community' or 'enterprise'."
+  }
+}
+
+variable "airbyte_helm_version" {
+  description = "The version of the Airbyte Helm chart to deploy"
+  type        = string
+  default     = "1.3.1"
+}
+
+variable "airbyte_version" {
+  description = "The version of Airbyte to deploy (for image caching)"
+  type        = string
+  default     = "1.3.1"
+}
+
+variable "domain" {
+  description = "The domain to access Airbyte (e.g., airbyte.example.com)"
+  type        = string
+}
+
+variable "wait" {
+  description = "Whether to wait for resources to be created before completing"
+  type        = bool
+  default     = true
+}
+
+variable "sla_target" {
+  description = "SLA target level (1-3) affecting high availability settings"
+  type        = number
+  default     = 1
+  validation {
+    condition     = var.sla_target >= 1 && var.sla_target <= 3
+    error_message = "The sla_target value must be between 1 and 3."
+  }
+}
+
+variable "admin_email" {
+  description = "Email for the admin user when auth is enabled"
+  type        = string
+}
+
+variable "ingress_enabled" {
+  description = "Whether to enable the ingress for Airbyte"
+  type        = bool
+  default     = true
+}
+
+variable "connector_builder_enabled" {
+  description = "Whether to enable the connector builder server"
+  type        = bool
+  default     = true
```

Comment:
false

Reply:
I am removing this var as it is required for airbyte to boot and function. I've attempted with it false and it'll result in a crash loop due to it not being available. 

https://github.com/airbytehq/airbyte/issues/25174